README.md

```diff
@@ -1,7 +1,44 @@
 # loopback-context
 
 Current context for LoopBack applications, based on
-node-continuation-local-storage.
+cls-hooked.
+
+## USAGE WARNING
+
+**Only if you use this package, do NOT run your app using `slc run` or `node .`**
+
+Run using:
+
+`node -r cls-hooked .`
+
+This uses the `-r` option in order to require `cls-hooked` before your app (see warnings below for more info).
+
+If you wish to use `strong-supervisor`, you would need to pass node options to `slc run`, which currently has issues, according to [strong-supervisor#56](https://github.com/strongloop/strong-supervisor/issues/56).
+
+## INSTALL WARNING
+
+**Only if you use this package, do NOT install your app using `npm install`.**
+
+Install using:
+
+```
+npm config set engine-strict true
+npm install
+```
+
+This keeps you from using Node < v4.5.
+
+## TEST WARNING
+
+**Do NOT test this package using `mocha .`.**
+
+Test using:
+
+```
+npm test
+```
+
+This adds the `-r` option to `mocha` command, needed in order to pass tests.
```

Comment:
Please move the `-r` option to `test/mocha.opts` file, so that people can run tests using `mocha` directly. I personally use this frequently, e.g. to run only a single test via `mocha -g "test name"`.

```
--require cls-hooked
```

(BTW you pull request does not change `package.json` despite what README says.)